index.html

```diff
@@ -9,24 +9,71 @@
   <body>
     
      <div class="wrapper">
-      <h3>All Awesome Books</h3> 
-      <!---<div class="container-book"></div>--> 
-      <div class="book-from-storage"> </div> 
+        <header>
+          <nav class="menu-bar">
+            <li class="logo">
+              <a href="#" class="menu-item-link">
+                Awesome Books
+              </a>
+            </li>
+            <ul class="menus">
+              
+              <li class="menu-item">
+                <a href="#list" class="menu-item-link list-anchor">
+                  List
+                </a>
+              </li>
+              <li class="menu-item">
+                <a href="#add-new" class="menu-item-link add-new-anchor">
+                  Add new
+                </a>
+              </li>
+              <li class="menu-item">
+                <a href="#contact" class="menu-item-link contact-anchor">
+                  Contact
+                </a>
+              </li>           
 
-      <div class="hor-line"></div>
+            </ul>
+          </nav>
+        </header> 
```

Comment:
- [ ] I have realized that there is some bit missing in your design that is in the wireframe provided. Kindly add the separator bars so that your design matches the provided wireframe.

**This is your design:**

![image](https://user-images.githubusercontent.com/86318284/193426725-f92250ec-67d7-4509-8ace-08d13d88b49e.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/193426749-a49997f5-1da4-4e54-a3f1-e2509280fb67.png)